src/PhpSpreadsheet/Calculation/LookupRef/Indirect.php

```diff
@@ -2,45 +2,75 @@
 
 namespace PhpOffice\PhpSpreadsheet\Calculation\LookupRef;
 
+use Exception;
 use PhpOffice\PhpSpreadsheet\Calculation\Calculation;
 use PhpOffice\PhpSpreadsheet\Calculation\Functions;
 use PhpOffice\PhpSpreadsheet\Cell\Cell;
 use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
 
 class Indirect
 {
+    /**
+     * Determine whether cell address is in A1 (true) or R1C1 (false) format.
+     *
+     * @param null|bool|float|int|string $a1fmt
```

Comment:
If you're listing all these potential parameter values, it's best to use `mixed` (and in various other method docblocks as well)

Reply:
Done.